.github/workflows/CI.yml

```diff
@@ -0,0 +1,82 @@
+name: CI
+# Run on tags or any pull request
+on:
+  push:
+    tags: ["*"]
+  pull_request:
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.version == 'nightly' }}
```

Comment:
This doesn't currently work the way we want so maybe we should just stop testing on julia nightly unless we're okay with spam.

Reply:
yeah continue on error is only used when fail fast is set to true. Potentially we just don't test master on nightly at all until the allow failure is fixed